requirements.txt

```diff
@@ -11,8 +11,8 @@ git+https://github.com/dwf/picklable_itertools.git#egg=picklable-itertools
 
 --allow-external theano
 --allow-unverified theano
-git+https://github.com/Theano/Theano.git#egg=theano
+git+https://github.com/Theano/Theano.git@c6d2ab3929182cdf7cf12e4b1f191a6ff99d07cc#egg=theano
```

Comment:
Right, except in this case the hotfix will only be applied to the `stable` branch, which will probably just give rise to merge conflicts once we perform an actual hotfix :p This two-branch thing is going to be fun!